pkg/arvo/sys/vane/gall.hoon

```diff
@@ -2,1457 +2,2882 @@
-  ==                                                    ::
-::                                                      ::
-::  XX a hack, required to break a subscription loop    ::
-::  which arises when an invalid mark crashes a diff.   ::
-::  See usage in ap-misvale.                            ::
-++  misvale-data  (set wire)                            ::  subscrs w/ bad marks
-++  seat                                                ::  agent state
-  $:  misvale/misvale-data                              ::  bad reqs
-      vel/worm                                          ::  cache
-      arms=(map [term path] (unit (pair @ud term)))     ::  ap-find cache
-      mom/duct                                          ::  control duct
-      liv/?                                             ::  unstopped
-      toc/torc                                          ::  privilege
-      tyc/stic                                          ::  statistics
-      ged/gest                                          ::  subscribers
-      hav/vase                                          ::  running state
-      byk/beak                                          ::  update control
-      pyl/(map bone mark)                               ::  req'd translations
-      zam/scar                                          ::  opaque ducts
-  ==                                                    ::
-++  sofa                                                ::  queue for blocked
-  $:  kys/(qeu (trel duct prey club))                   ::  queued kisses
-  ==                                                    ::
-++  stic                                                ::  statistics
-  $:  act/@ud                                           ::  change number
-      eny/@uvJ                                          ::  entropy
-      lat/@da                                           ::  time
-  ==                                                    ::
---                                                      ::
-    ::::::::::::::::::::::::::::::::::::::::::::::::::::::  vane header
-    ::::::::::::::::::::::::::::::::::::::::::::::::::::::
+::
+::  (rest of arvo)
+::
+|%
+::
+::  +internal-gift: synonym for +cuft.
+::
+++  internal-gift  cuft
+::
+::  +internal-task: synonym for +cush.
+::
+++  internal-task  cush
+::
+::  +agent-action: synonym for +club.
+::
+++  agent-action  club
+::
+::  +coke: cook.
+::
+++  coke
+  $?  %inn
+      %out
+      %cay
+  ==
+::
+::  +volt: voltage.
+::
+++  volt  ?(%low %high)
+::
+::  +security-control: security control.
+::
+++  security-control  $@(?(%iron %gold) [%lead p=ship])
+::
+::  +reverse-ames: reverse ames message.
+::
+++  reverse-ames
+  $%
+      :: diff
+      ::
+      [action=%d p=mark q=*]
+      ::  etc.
+      ::
+      [action=%x ~]
+  ==
+::
+::  +forward-ames: forward ames message.
+::
+++  forward-ames
+  $%
+      :: message
+      ::
+      [action=%m mark=mark noun=*]
```

Comment:
for consistency, we should use the `=mark` syntax here, and in other type definitions